packages/docs/src/components/Sections/Overview/OverviewItem/index.tsx

```diff
@@ -18,19 +19,25 @@ const OverviewItem = ({ label, title, body, mediaSrc }: Props) => {
     <>
       <section className={styles.description}>
         <p className={styles.itemLabel}>{label}</p>
-        <h2 className={styles.itemTitle}>{title}</h2>
+        <h3 className={styles.itemTitle}>{title}</h3>
         <p className={styles.itemBody}>{body}</p>
         <LinkButton
           title="Get started for free"
           href="https://marketplace.visualstudio.com/items?itemName=swmansion.react-native-ide"
           onClick={handleButtonClick}
         />
       </section>
-      <div className={styles.media}>
-        <video autoPlay loop muted playsInline>
-          <source src={mediaSrc} type="video/mp4" />
-        </video>
-      </div>
+      <InView triggerOnce>
```

Comment:
Could you add to the PR description how this helps? Some comparison, so in the future we remember why another package actually helped with performance which might be counterintuitive :D

Reply:
sure

 i'm using intersection observer to load videos only when they are in view. it doesn't make sense to load 10MB of videos for every user even when they never scroll down the page
 
I show a lightweight (~7kb) placeholder image during the video loading